adagrams/submit.md

```diff
@@ -83,3 +83,48 @@ The URL to your team pull request: https://github.com/<some-ada-repo>/<project-n
 ##### !end-answer
 ### !end-challenge
 <!-- prettier-ignore-end -->
+
+## Tool Usage Reflection
+
+<!-- prettier-ignore-start -->
+### !challenge
+* type: paragraph
+* id: 0a403185-cde2-4a5b-9bfb-7333ff3db96h
+* title: Tool Usage Reflection
+##### !question
+
+If you used AI tools while working on this project, please share links to the AI chats you used to discuss the project and answer the following questions:
```

Comment:
I don't think what I'm about to suggest is **not** important to the change, but more so for organization! How would we feel about a separate submission box for the AI chat link? I don't think that would be a huge issue to keep it them together, but it would be a small nicety for us to know it would always be in the same spot. Let me know what you think!

Reply:
I'm good breaking that into a separate box and asking them to use a space to separate links if there are more than one 👍

Reply:
I'm good with that. I had an initial thought to keep them together in case someone didn't use any AI they wouldn't have multiple things to add "N/A" to, but if it might be easier for instructors, I'm happy to split those out.